draft-ietf-tls-tls13.md

```diff
@@ -2244,16 +2244,18 @@ SignatureSchemeList value:
            ed25519(0x0807),
            ed448(0x0808),
 
+           /* Legacy algorithms */
+           ecdsa_sha1(0x0203),
```

Comment:
Yeah, it probably makes sense to call rsa_pkcs1_sha1 a legacy algorithm, even though it's sufficiently entrenched that we will still be using it for the near future.